resources/2019/views/partials/workshop.blade.php

```diff
@@ -1,10 +1,10 @@
 <div class="md:flex mb-6">
     <div class="md:w-1/6 text-center">
         <a href="{{ $speaker->url() }}">
-            <div class="speaker enlarge image-border-gradient-left w-1/2 inline-block shadow-lg-blue rounded max-h-32 md:w-32 overflow-hidden">
+            <div class="speaker enlarge image-border-gradient-left w-1/2 inline-block shadow-lg-blue rounded h-32 md:w-32 overflow-hidden">
                 <div style="height: inherit; transform: translateY(50%);">
-                    <img src="{{ $speaker->photoUrl() }}" alt="{{ $speaker->name }}"
-                         class="block w-full" style="transform: translateY(-50%);">
+                    <img src="{{ str_contains($speaker->photo, 'cloudinary') ? $speaker->photo : $speaker->photoUrl() }}" alt="{{ $speaker->name }}"
```

Comment:
The feed contains workshops from 2019 as well, this is a fix for the 2 that weren't on the site yet.